src/main/java/frc/robot/controls/BaseDriverConfig.java

```diff
@@ -10,7 +10,7 @@
  */
 public abstract class BaseDriverConfig {
 
-    private final Drivetrain drive;
+    protected final Drivetrain drive;
```

Comment:
Yeah, so what protected final does is is that it's accessible within the class and its subclasses, whereas private final means the field is only accessible within the class itself. So with protected final you can have subclass access while still stopping external access from outside the class, while private final restricts both subclass and external access. We typically use private because we want to limit access strictly to the defining class, since generally there's better encapsulation—it's kept internal to the class and can’t be changed by subclasses or external code.

Reply:
fixed it!